fitstack/signal.py

```diff
@@ -1086,6 +1109,22 @@ def _interpret_ps2Ds(self, ps2Ds: Dict[str, Powerspec1D]):
             scale = self._solve_scale(base, ps2Ds[key], alpha)
             self._convolution_scale[name] = scale
 
+
+    def _get_factor(self) -> float:
+        """Calculate the conversion factor connecting tau and k_parallel.
+        
+        C = -1/(2 pi nu21) * c/H(z) * (1+z)²
+        
+        Returns
+        -------
+        C : float
+            Conversion factor
+        """
+
+        C = (-1.0 / (2 * np.pi * self.nu21)) * (self.cs / self.H_z) * (1 + self.z_eff)**2
```

Comment:
```suggestion
        C = (-1.0 / (2 * np.pi * u.nu21)) * ((u.c / u.kilo) / self.H_z) * (1 + self.z_eff)**2
```
Use `cora`'s `nu21` and speed of light.

Also, let's add a comment about the assumed units of `k_parallel` and `tau` that this factor is converting between.